controllers/BookmarksController.js

```diff
@@ -0,0 +1,121 @@
+
+import Models from '../models';
+import StatusResponse from '../helpers/StatusResponse';
+
+const { bookmarks, articles } = Models;
+/** @description usersController class
+ * @return {object} the response object
+ * @public
+ */
+class BookmarksController {
+  /** @description create a bookmark
+   * @param {string} req is the request parameter
+   * @param {string} res is the response parameter
+   * @return {object} the response object
+   * @public
+   */
+  static async create(req, res) {
+    const { userId } = res.locals.user;
+    const { articleId } = req.params;
+    let { title } = req.body;
+
+    if (!title) {
+      title = `untitled ${articleId}`;
+    }
+
+    try {
+      const bookmark = await bookmarks.create({
+        title,
+        userId,
+        articleId
+      });
+
+      if (!bookmark) {
+        return StatusResponse.badRequest(res, { message: 'error posting bookmark' });
+      }
+
+      return StatusResponse.success(res, { message: 'hurray! bookmark was added successfully' });
+    } catch (error) {
+      return StatusResponse.internalServerError(error);
+    }
+  }
+
+  /** @description get one bookmark
+   * @param {object} req is the request parameter
+   * @param {object} res is the response parameter
+   * @return {object} the response object
+   * @public
+   */
+  static async search(req, res) {
+    const { title } = req.body;
+    const { userId } = res.locals.user;
+
+    if (!title) {
+      return StatusResponse.badRequest(res, { message: 'please insert a title' });
+    }
+
+    const bookmark = await bookmarks.findAll({
+      where: {
+        title,
+        userId,
+      },
+      include: [{
+        model: articles,
+        as: 'article',
+        attributes: { exclude: ['createdAt', 'updatedAt', 'slug'] }
+      }],
+    });
+    if (!bookmark) {
+      return StatusResponse.notfound(res, { message: 'no bookmark found' });
+    }
+    return StatusResponse.success(res, bookmark);
```

Comment:
The search result should be paginated.
@pitaz What do you think here? Should we merge and add pagination later or wait for your helper to be created first?

Reply:
@Chidi3b3r3 We can go ahead and merge. Pagination will be added later in my next PR.